cmd/dep/status.go

```diff
@@ -261,7 +261,8 @@ func runStatusAll(ctx *dep.Ctx, out outputter, p *dep.Project, sm gps.SourceMana
 	var digestMismatch, hasMissingPkgs bool
 
 	if p.Lock == nil {
-		return digestMismatch, hasMissingPkgs, errors.Errorf("no Gopkg.lock found. Run `dep ensure` to generate lock file")
+		// TODO if we have no lock file, do...other stuff
+		return digestMismatch, hasMissingPkgs, nil
```

Comment:
this also indicates a detailed review of the changes to existing files is necessary - i may have unintentionally overwritten some other changes, too.